src/views/pacbio/PacbioLibraryBatchCreate.vue

```diff
@@ -94,6 +94,16 @@
                 <traction-table :fields="state.csvTableFields" :items="state.csvData" />
               </div>
             </div>
+            <div class="flex flex-row px-4 space-x-2 h-2">
+              <div
+                v-if="isCreationInProgress"
+                id="progress-indicator"
+                class="flex flex-row items-center"
+              >
+                <label class="text-md text-blue-400">Creating Libraries...</label>
```

Comment:
Advisory: In other places we put the spinner in the button - e.g. https://github.com/sanger/traction-ui/blob/develop/src/components/pacbio/PacbioPoolEdit.vue#L110. Might be slightly clearer if we do the same here.

Reply:
Good point! I missed that. It's better to keep it consistent. 👍

Reply:
Do you think we still need the `Creating libraries..` text too? Not sure if that can be removed or moved to the button if the spinner is enough indication.

Reply:
I thought the 'Creating libraries...' text would make it clearer to users what action is happening as though the spinner is a great visual indicator, it doesn’t explicitly tell users what is being processed.
Moving the text into the button would mean replacing the button's label dynamically, which I am not much into as the same button displaying two different button texts which serves different purposes.
Do you think the spinner is good enough, then I will remove text?